barter-data/src/provider/databento/mod.rs

```diff
@@ -0,0 +1,33 @@
+use barter_instrument::Side;
+use databento::dbn::Side as DbSide;
+use serde::{Deserialize, Serialize};
+
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Deserialize, Serialize)]
+pub enum DatabentoSide {
+    #[serde(alias = "buy", alias = "BUY", alias = "b")]
+    Buy,
+    #[serde(alias = "sell", alias = "SELL", alias = "s")]
+    Sell,
+}
+
+
+impl From<DbSide> for DatabentoSide {
+    fn from(value: DbSide) -> Self {
+        match value {
+            DbSide::Bid => DatabentoSide::Buy,
+            DbSide::Ask => DatabentoSide::Sell,
+            _ => {
+                panic!("Invalid side")
+            }
+        }
+    }
+}
+
+impl Into<Side> for DatabentoSide {
```

Comment:
Always implement From instead of Into. 

See https://doc.rust-lang.org/std/convert/trait.From.html

```
One should always prefer implementing From over [Into](https://doc.rust-lang.org/std/convert/trait.Into.html) because implementing From automatically provides one with an implementation of [Into](https://doc.rust-lang.org/std/convert/trait.Into.html) thanks to the blanket implementation in the standard library.
```